### ISSUE TITLE
It'd be nice to see config errors

### ISSUE THREAD

**Issue by bytenel** [sirlantis/rubocop-for-rubymine#18]
When running rubocop manually for one of my projects I get an error:

```
... unrecognized parameter AllCops:Style/CaseIndentation ...
```
Which would have made debugging what was happening in the plugin much easier to figure out. I was frustrated because I kept getting a problem in one of my files but it seems that my config file was causing the problem. It would have been nice to get an alert for that.

**Comment by sirlantis** [comment]
Apparently printed to STDERR in newer RuboCop versions:

    Warning: unrecognized cop Style/CaseIndentation found in /.../.rubocop.yml`

**Comment by sirlantis** [comment]
Fixed by 602e0dff09989c512f91979d37e3d66b1606989e - not released yet. Probably fragile.
